packages/ddp-server/livedata_server.js

```diff
@@ -1705,9 +1762,18 @@ Object.assign(Server.prototype, {
     }
   },
 
-  _removeSession: function (session) {
+  _removeSession: function (session, callback = () => {}) {
     var self = this;
-    self.sessions.delete(session.id);
+    const f = () => {
```

Comment:
The name of this function could be better, for debugging reasons :)

Reply:
Renamed to `sessionRemoveFunction`